libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -520,6 +520,35 @@ - (SFRestRequest *)cheapRequest:(NSString *)apiVersion {
     return [self requestForResources:apiVersion];
 }
 
+- (SFRestRequest *)requestForNotificationTypes {
```

Comment:
LGTM - could you add some tests in https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/dev/libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceRestAPITests.m for the new methods?